components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/util/OAuthLogEventConstants.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023, WSO2 Inc. (https://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.identity.oauth.util;
+
+/**
+ * This class contains the constants related to OAuth Logging.
+ */
+public class OAuthLogEventConstants {
+
+    /**
+     * Enum for OAuth Event Categories.
+     */
+    public static enum EventCatalog {
+        CREATE_OAUTH_APPLICATION("CREATE OAUTH APPLICATION", "Create OAuth Application"),
+        UPDATE_OAUTH_APPLICATION("UPDATE OAUTH APPLICATION", "Update OAuth Application"),
+        DELETE_OAUTH_APPLICATION("DELETE OAUTH APPLICATION", "Delete OAuth Application"),
+        DELETE_OAUTH_APPLICATIONS_IN_THE_GIVEN_ORG("DELETE OAUTH APPLICATION IN GIVEN ORG",
+                "Delete OAuth Application in the given org"),
+        REGENERATE_CLIENT_SECRET("REGENERATE CLIENT SECRET", "Regenerate client secret of OAuth Application"),
+        UPDATE_APP_STATE("UPDATE APP STATE", "Update OAuth Application State");
+
+        private final String eventId;
+        private final String eventDescription;
+
+        private EventCatalog(String eventId, String eventDescription) {
+            this.eventId = eventId;
```

Comment:
missing new line after the method signature 
check other places as well

Reply:
Removed this class and moved to string constant. Using enum might give some additional restrictions. In future, we need to have an osgi service to register all the suppported events. Using that, we can have some REST API to get the supported events